shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -297,6 +297,10 @@ public final class MetricsNames {
     public static final String SEAL_STREAM_FAILED = PREFIX + "controller.stream.seal_failed";                // Counter and Per-stream Counter
     public static final String DELETE_STREAM = PREFIX + "controller.stream.deleted";                         // Counter
     public static final String DELETE_STREAM_LATENCY = PREFIX + "controller.stream.deleted_latency_ms";      // Histogram
+    public static final String DELETE_STREAM_EVENT_LATENCY = PREFIX + "controller.stream.event.deleted_latency_ms";      // Histogram
```

Comment:
These metrics seem misleading: what you are measuring the time of posting events for Controller EventProcessors. But the name of the metric may confuse people creating dashboards out of it, as they may get confused that it reports the write or seal latency for a stream `controller.stream.event.sealed_latency_ms`. 

My recommendation is to use a block of metrics for this purpose only, like:
`controller.eventProcessors.deleteStream.write_latency_ms`
`controller.eventProcessors.updateStream.write_latency_ms`
`controller.eventProcessors.sealStream.write_latency_ms`
`controller.eventProcessors.truncateStream.write_latency_ms`

Reply:
updated names as suggested